dc-model/src/main/java/de/digitalcollections/model/filter/FilterCriterion.java

```diff
@@ -7,9 +7,12 @@
  * Filter Criterion Container.A filter criterion is a composition of
  *
  * <ul>
- *   <li>a field name being target of filter operation
+ *   <li>an expression (e.g. field name) being target of filter operation
  *   <li>a filter operation (operator)
  *   <li>one ore more filter values to be used for filtering operation (operand(s))
+ *   <li>a flag to indicate if expression has to be nativeExpression or be used unchanged in
+ *       underlying backend system (e.g. mapping Java member names to database column names; no
```

Comment:
```suggestion
 *       underlying backend system (e.g. mapping Java member names to database column names vs. no
```
Just to point it out more precisely